README.md

```diff
@@ -4,9 +4,11 @@
 
 *Warning*: currently pre-release and subject to frequent breaking changes. Use at your own risk.
 
-Grafana JavaScript Agent is a library that enables applications (both frontend and backend) to interact with external
-services like [Grafana](https://grafana.com/grafana/), [Prometheus](https://prometheus.io/),
-[Loki](https://grafana.com/logs/), [Tempo](https://grafana.com/traces/) etc.
+Grafana JavaScript Agent is a library that can instrument javascript applications
+(both frontend and (soon!) backend) to collect telemetry and forward it
+to the [Grafana Agent](https://grafana.com/docs/agent/latest/)
+(with app o11y collector integration enabled). Grafana Agent then can send this data further to [Prometheus](https://prometheus.io/),
```

Comment:
We should adopt that feature of Markdown where we set links in the footer of the document and the link them inline. This will allow us to point to the `Grafana Agent` (for example) page everywhere instead of one time only (as a developer I hate when i have to scroll to look for a clickable term 😢 ).

Reply:
oh, didn't know this was a thing
- [ ] footer links of the document

Reply:
I think it's just a preference. I prefer inline links and that's what the docs team mostly uses as best practice, but always open to new ways :)

Reply:
I think it's just a preference. I prefer inline links and that's what the docs team mostly uses as best practice, but always open to new ways :)

Reply:
So add normal link on every mention, or keep as is - just the first one, wdyt @brendamuir?

Reply:
If it's multiple mentions in the same topic, just link from the first one

Reply:
ok, then lets keep as is